com.github.muriloventuroso.easyssh.json

```diff
@@ -0,0 +1,90 @@
+{
+    "app-id": "com.github.muriloventuroso.easyssh",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.26",
+    "sdk": "org.gnome.Sdk",
+    "command": "com.github.muriloventuroso.easyssh",
+    "finish-args": [
+        "--share=ipc",
```

Comment:
I assume it needs at least networking access and read-only to SSH keys. I'm not sure if it's enough to actually make ssh happy – which by the way is not bundled.